libraries/dialogs/src/main/java/com/trendyol/uicomponents/dialogs/DialogViewState.kt

```diff
@@ -21,6 +23,12 @@ data class DialogViewState(
     fun getContent(): Spanned = if (showContentAsHtml) {
```

Comment:
Return type getContent() functions can be replaced with CharSequence to avoid using a SpannableString in else statement.

Reply:
`HtmlCompat.fromHtml()` method returns Spanned.